src/test/java/com/woowacamp/soolsool/integration/repository/CartItemRepositoryTest.java

```diff
@@ -91,4 +92,121 @@ void createCartItem() {
         // then
         assertThat(saved.getId()).isNotNull();
     }
+
+    @Test
+    @DisplayName("유저 아이디에 따른 장바구니 모두 조회")
+    void cartItemListByUserId() {
+        LiquorBrew liquorBrew = liquorBrewRepository.findById(1L)
+            .orElseThrow(() -> new RuntimeException("LiquorBrew가 존재하지 않습니다."));
+        LiquorRegion liquorRegion = liquorRegionRepository.findById(1L)
+            .orElseThrow(() -> new RuntimeException("LiquorRegion이 존재하지 않습니다."));
+        LiquorStatus liquorStatus = liquorStatusRepository.findById(1L)
+            .orElseThrow(() -> new RuntimeException("LiquorStatus가 존재하지 않습니다."));
+
+        Liquor liquor = Liquor.builder()
+            .brew(liquorBrew)
+            .region(liquorRegion)
+            .status(liquorStatus)
+            .name("안동 소주")
+            .price("12000")
+            .brand("안동")
+            .imageUrl("/soju.jpeg")
+            .stock(120)
+            .alcohol(21.8)
+            .volume(400)
+            .build();
+        liquor = liquorRepository.save(liquor);
+        Liquor liquor2 = Liquor.builder()
+            .brew(liquorBrew)
+            .region(liquorRegion)
+            .status(liquorStatus)
+            .name("새로")
+            .price("12000")
+            .brand("새로")
+            .imageUrl("/sero.jpeg")
+            .stock(120)
+            .alcohol(21.8)
+            .volume(400)
+            .build();
+        liquor2 = liquorRepository.save(liquor2);
+
+        CartItem cartItem = CartItem.builder()
+            .memberId(commonMemberId)
+            .liquor(liquor)
+            .quantity(1)
+            .build();
+        cartItemRepository.save(cartItem);
+        CartItem cartItem2 = CartItem.builder()
+            .memberId(commonMemberId)
+            .liquor(liquor2)
+            .quantity(1)
+            .build();
+        cartItemRepository.save(cartItem2);
+
+        // when
+        List<CartItem> cartItemList = cartItemRepository
+            .findAllByMemberIdOrderByCreatedAtDescWithFetchJoin(commonMemberId);
+        
+        // then
+        assertThat(cartItemList).hasSize(2);
+    }
```

Comment:
```suggestion
    @Test
    @DisplayName("유저 아이디에 따른 장바구니 모두 조회")
    void cartItemListByUserId() {
	// given

        // when
        List<CartItem> cartItemList = cartItemRepository
            .findAllByMemberIdOrderByCreatedAtDescWithFetchJoin(commonMemberId);
        
        // then
        assertThat(cartItemList).hasSize(2);
    }
```

Repository 통합 테스트를 인수 테스트처럼 save하고 할 것인가, 아니면 data.sql에 의존할 것인가에 대해 의논해보는 것도 좋을 것 같아요.